README.md

```diff
@@ -11,6 +11,7 @@ It works like excl
 and this is a cat
```

Comment:
This is cute, but now that you're about to finalize the code, change it to something related to your app. (Or leave it blank)